scripts/setup-minikube-for-linux.sh

```diff
@@ -2,8 +2,9 @@
 
 set -exu
 
-KUBERNETES_VERSION="v1.15.2"
-MINIKUBE_VERSION="v1.2.0"
+# Keep it in sync with the version declared in scripts/setup-tools-for-debian.sh
+KUBERNETES_VERSION="v1.18.18"
+MINIKUBE_VERSION="latest"
```

Comment:
You sure you want this to be a moving target?

Reply:
Yes, that is that we are doing for lightbend/console-gui already, which uses this reposity in its build. So at least it is consistent.